docs/source/developers/cpp/building.rst

```diff
@@ -39,8 +39,8 @@ out-of-source. If you are not familiar with this terminology:
 
 Building requires:
 
-* A C++11-enabled compiler. On Linux, gcc 4.8 and higher should be
-  sufficient. For Windows, at least Visual Studio 2017 is required.
+* A C++17-enabled compiler. On Linux, gcc TODO and higher should be
+  sufficient. For Windows, at least Visual Studio TODO is required.
```

Comment:
There's a bunch of information regarding this in https://docs.scipy.org/doc/scipy-1.9.1/dev/toolchain.html#c-language-standards (not the [dev](https://scipy.github.io/devdocs/dev/toolchain.html) version because main branch already moved to C++17 and removed some of that info), in short:
```suggestion
* A C++17-enabled compiler. On Linux, gcc 7.1 and higher should be
  sufficient. For Windows, at least Visual Studio VS2017 is required.
```

Reply:
Thanks for this. Will commit it.